pipelines/broad/internal/rna_seq/BroadInternalRNAWithUMIs.changelog.md

```diff
@@ -3,6 +3,7 @@
 
 * Updated SortSam task to be able to use call caching
 * Updated STAR aligner to 2.7.10a for RNAWithUMIs pipeline
+* Added optional tasks to write outputs to the Terra Data Repository
```

Comment:
nit: Date of Last Commit (line 2) should be: 2022-01-31

Reply:
nice catch, thanks!